### PR TITLE
feat: Solana bond

### PR DESCRIPTION
**Pull request by mohandast52**
- A thorough review is needed for the dummy bond calculation for the SVM product.

https://github.com/valory-xyz/autonolas-tokenomics-frontend/assets/22061815/9122859d-4076-498a-a9ce-32043f73cd72

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -28,6 +28,7 @@
     "@wagmi/core": "^2.6.16",
     "@web3modal/wagmi": "^4.1.6",
     "antd": "^5.9.0",
+    "bignumber.js": "^9.1.2",
```
Comment (truemiller) [inline package.json:31]: why do we need another bignumber package?
ethers and web3 both have them, they use bn.js under the hood

Comment (truemiller) [reply package.json:31]: Sure, can understand. 

Just be mindful that there may be incompatibility between bn.js and bignumber.js.

And we should never send a float to an EVM contract.

**Review by truemiller — DISMISSED** [review]

**Review by kupermind — APPROVED** [review]

**Review by oaksprout — APPROVED** [review]
Looks fine from a product perspective, as you say – requires a thorough review from protocol folks.

**Review by mariapiamo — APPROVED** [review]

File: components/Home/Bonding/BondingList/useBondingList.jsx [hunk 2]
```diff
@@ -393,9 +393,7 @@ const useAddProjectChangeToProducts = () =>
   useCallback(
     (productList) =>
       productList.map((record) => {
-        // current price of the LP token is multiplied by 2
-        // because the price is for 1 LP token and
-        // we need the price for 2 LP tokens
+        // To calculate the price of LP we need to multiply the price by 2
```
Comment (mariapiamo) [inline components/Home/Bonding/BondingList/useBondingList.jsx:403]: // To calculate the price of LP we need to multiply olasReserve/TotalSupply by 2

**Review by mariapiamo — APPROVED** [review]

**Review by truemiller — APPROVED** [review]

**Review by DavidMinarsch — APPROVED** [review]

**Review by mariapiamo — APPROVED** [review]

**Pull request merged**